### PR TITLE
chore(protocol): remove code related to PSE zkEVM

### PR DESCRIPTION
**Pull request by dantaik**

### CODE REVIEW

File: packages/protocol/contracts/L1/TaikoL1.sol [hunk 1]
```diff
@@ -210,7 +210,7 @@ contract TaikoL1 is EssentialContract, ITaikoL1, ITierProvider, TaikoEvents, Tai
     function getConfig() public view virtual override returns (TaikoData.Config memory) {
         // All hard-coded configurations:
         // - treasury: the actual TaikoL2 address.
-        // - blockMaxTxs: 80 (limited by the PSE zkEVM circuits)
+        // - blockMaxTxs: 80
```
Comment (davidtaikocha) [inline packages/protocol/contracts/L1/TaikoL1.sol:213]: Then shall we just remove `blockMaxTxs` + `blockMaxGasLimit` limits from our protocol?  or even allow a larger `blockMaxGasLimit` than 30m?

Comment (dantaik) [reply packages/protocol/contracts/L1/TaikoL1.sol]: blockMaxTxs was removed already. I shall delete this whole line

Comment (dantaik) [reply packages/protocol/contracts/L1/TaikoL1.sol]: Regarding blockMaxGasLimit I'm not sure, maybe we keep it for now as we'll use zkVM as well.

**Review by davidtaikocha — APPROVED** [review]

**Review by Brechtpd — APPROVED** [review]

File: packages/protocol/contracts/L1/TaikoL1.sol (hunk 1 above)
Comment (Brechtpd) [reply packages/protocol/contracts/L1/TaikoL1.sol]: A higher `blockMaxGasLimit` I think would be very beneficial to reduce onchain costs, but if we do it I think we first have to test it out how the node/provers behave under big blocks.

**Pull request merged**